src/routes/Home.js

```diff
@@ -286,137 +267,91 @@ const ReadOnly = styled('span')`
   margin-left: 1em;
 `
 
+export const HOME_DATA = gql`
```

Comment:
Previously all gql was in on its own file like `queries.js`. Are we now start embedding at routes and component level?